docs/source/api/v2/servers_id_deliveryservices.rst

```diff
@@ -67,78 +67,78 @@ Request Structure
+:matchList:			The :term:`Delivery Service`'s :ref:`ds-matchlist`
+
+	:pattern:	A regular expression - the use of this pattern is dependent on the ``type`` field (backslashes are escaped)
+	:setNumber:	An integer that provides explicit ordering of :ref:`ds-matchlist` items - this is used as a priority ranking by Traffic Router, and is not guaranteed to correspond to the ordering of items in the array.
+	:type:		The type of match performed using ``pattern``.
+
+:maxDnsAnswers:		The :ref:`ds-max-dns-answers` allowed for this :term:`Delivery Service`
+:maxOriginConnections:	The :ref:`ds-max-origin-connections`
+:midHeaderRewrite:	A set of :ref:`ds-mid-header-rw-rules`
+:missLat:		The :ref:`ds-geo-miss-default-latitude` used by this :term:`Delivery Service`
+:missLong:		The :ref:`ds-geo-miss-default-longitude` used by this :term:`Delivery Service`
+:multiSiteOrigin:	A boolean that defines the use of :ref:`ds-multi-site-origin` by this :term:`Delivery Service`
+:orgServerFqdn:		The :ref:`ds-origin-url`
+:originShield:		A :ref:`ds-origin-shield` string
+:profileDescription:	The :ref:`profile-description` of the :ref:`ds-profile` with which this :term:`Delivery Service` is associated
+:profileId:		The :ref:`profile-id` of the :ref:`ds-profile` with which this :term:`Delivery Service` is associated
+:profileName:		The :ref:`profile-name` of the :ref:`ds-profile` with which this :term:`Delivery Service` is associated
+:protocol:		An integral, unique identifier that corresponds to the :ref:`ds-protocol` used by this :term:`Delivery Service`
+:qstringIgnore:		An integral, unique identifier that corresponds to the :ref:`ds-qstring-handling` setting on this :term:`Delivery Service`
+:rangeRequestHandling:	An integral, unique identifier that corresponds to the :ref:`ds-range-request-handling` setting on this :term:`Delivery Service`
+:regexRemap:		A :ref:`ds-regex-remap`
+:regionalGeoBlocking:	A boolean defining the :ref:`ds-regionalgeo` setting on this :term:`Delivery Service`
+:remapText:		:ref:`ds-raw-remap`
+:signed:		``true`` if  and only if ``signingAlgorithm`` is not ``null``, ``false`` otherwise
+:signingAlgorithm:	Either a :ref:`ds-signing-algorithm` or ``null`` to indicate URL/URI signing is not implemented on this :term:`Delivery Service`
+:sslKeyVersion:		This integer indicates the :ref:`ds-ssl-key-version`
+:tenantId:		The integral, unique identifier of the :ref:`ds-tenant` who owns this :term:`Delivery Service`
+:trRequestHeaders:	If defined, this defines the :ref:`ds-tr-req-headers` used by Traffic Router for this :term:`Delivery Service`
+:trResponseHeaders:	If defined, this defines the :ref:`ds-tr-resp-headers` used by Traffic Router for this :term:`Delivery Service`
+:type:			The :ref:`ds-types` of this :term:`Delivery Service`
+:typeId:		The integral, unique identifier of the :ref:`ds-types` of this :term:`Delivery Service`
+:xmlId:			This :term:`Delivery Service`'s :ref:`ds-xmlid`
```

Comment:
Ohhhhhh, I see what's happening. You're replacing all the spaces with tabs. But that destroys alignment; tabs are for indentation, spaces are for alignment.

Reply:
Aligned fields with spaces in c2b73f08e9.